### ISSUE TITLE
Screen reader reading "button" twice on multiple controls

### ISSUE THREAD

**Issue by dborovic-neogov** [videogular/ngx-videogular#138]
### Description
Screen reader reading "button" twice for all controls e.g. [https://github.com/videogular/ngx-videogular/blob/master/libs/ngx-videogular/controls/src/lib/components/vg-mute/vg-mute.component.ts](url). Issue is that control has role="button" (which is correct) and aria-label "mute button". aria-label should be "mute", otherwise screen readers read aria-label first and appends "button" after.

### Expected Behavior
When focusing any control element, screen reader should read e.g. "mute button". Should be solved by setting "aria-label" to just "mute" instead of "mute button".

### Actual Behavior
When focusing any control element, screen reader is reading e.g. "mute button button"

### Steps to Reproduce
Run any example, tested with NVDA on Windows. Open speech viewer and tab over controls

### Attachments
![image](https://github.com/videogular/ngx-videogular/assets/26741235/477a502f-940d-4af2-b59b-3b813a34dffa)
